pkgs/development/tools/build-managers/gup/default.nix

```diff
@@ -18,20 +18,23 @@ stdenv.mkDerivation rec {
     mkdir $out
     cp -r python/bin $out/bin
   '';
-  passthru.updateScript = ''
-    #!${runtimeShell}
-    set -e
-    echo
-    cd ${toString ./.}
-    ${nix-update-source}/bin/nix-update-source \
-      --prompt version \
-      --replace-attr version \
-      --set owner timbertson \
-      --set repo gup \
-      --set type fetchFromGitHub \
-      --set rev 'version-{version}' \
-      --modify-nix default.nix
-  '';
+  passthru.updateScript = [
+    runtimeShell
+    "-c"
+    ''
+      set -e
+      echo
+      cd ${toString ./.}
+      ${nix-update-source}/bin/nix-update-source \
+        --prompt version \
+        --replace-attr version \
+        --set owner timbertson \
+        --set repo gup \
+        --set type fetchFromGitHub \
+        --set rev 'version-{version}' \
+        --modify-nix default.nix
+    ''
+  ];
```

Comment:
This can most likely be replaced with nix-update.

Reply:
That sounds good, this update script is creaking at the seams. But that will need more work I'm afraid